Sources/AuthFoundation/Token Management/Token.swift

```diff
@@ -126,11 +128,8 @@ public final class Token: Codable, Equatable, Hashable, JSONClaimContainer, Expi
                 let request = Token.RefreshRequest(openIdConfiguration: configuration,
                                                    clientConfiguration: client.configuration,
                                                    refreshToken: refreshToken,
-                                                   id: Token.RefreshRequest.placeholderId,
-                                                   configuration: [
-                                                    "client_id": client.configuration.clientId,
-                                                    "scope": client.configuration.scopes
-                                                ])
+                                                   scope: nil,
```

Comment:
Why is `scope: nil`?

Reply:
Because it's inheriting the scopes from the initial token exchange.  But this argument is being exposed to the `refresh` function for future expansion, when I introduce support for downscoping refresh tokens.  I just didn't want to overly conflate this PR with additional user-facing features.